lldb/include/lldb/Symbol/CompilerType.h

```diff
@@ -436,8 +482,8 @@ class CompilerType {
                      ExecutionContextScope *exe_scope);
 
   /// Dump to stdout.
-  void DumpTypeDescription(lldb::DescriptionLevel level =
-                           lldb::eDescriptionLevelFull) const;
```

Comment:
I understand the desire to fix formatting issues, but let's avoid doing them in a big patch like this. If things needs to be reverted, as this patch was, such a formatting change should not need to be reverted.

This can also cause unnecessary work for downstream forks in case of merge conflicts (the first patch, the revert, the second patch, etc).

Reply:
Personally I agree with you. GIthub pull request creation did not give me any choice: It would not allow me to create a PR that did not pass clang-format. If you know of some way to bypass that requirement, PLEASE tell me!!

Reply:
> It would not allow me to create a PR that did not pass clang-format.

Can you be more specific here, you could not open a PR at all without it being formatted? If that's the case it doesn't seem like what anyone would have intended.

Your other workaround here would be to do a PR to reformat these files, then rebase this onto that after landing it. Review has already started though, so maybe do that as the last thing after approvals.

Reply:
DavidSpickett:  I created the PR, sort of, but... if you look at the bottom of this PR (for example) there's a small section that says "All Checks Passed". If you open that, you will see that one of the checks is a clang-format check.  My original PR did not pass that check, so github would not "finish" creating the PR, i.e. it would not allow me to ask for reviews on it.

Reply:
That's annoying for sure. I'm a member of the repo, and I was able to add reviewers to this test PR https://github.com/llvm/llvm-project/pull/7381 that failed clang-format. Then again, Github tells me you're also a member, so that can't be the problem. Strange.

If you can figure out the sequence of events or reproduce it, please open an issue because I'm not sure it's intended behaviour. Perhaps we're trying to be strict but still, there should be some way to ignore the check when needed.

Reply:
I could be wrong here, but AFAICT the bot should _only_ check for formatting changes inside the diff of the commit, not inside the entire file. If this is not what the bot is doing, it is something we can probably improve upon.

In the meantime though, my suggestion is to manually inspect the bot output: if it is complaining about lines not touched by this commit, just ignore that. This is favorable to editing lines that are not related to the patch

Reply:
I see this comment in [another PR](https://github.com/llvm/llvm-project/pull/73252#issuecomment-1832242278)

> NB: not applying all the clang-format recommendations as it affects lines I'm not editing, and I don't want to pollute git-blame

That makes me think the bot is not running clang-format-diff? https://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting

Reply:
Question:  For this PR, do you want me to un-do the clang-format changes that don't relate to my changes?

Reply:
Yes, please :)

Reply:
Done.